content/en/continuous_integration/setup_tests/python.md

```diff
@@ -39,36 +31,53 @@ pip install "ddtrace>=0.50.0rc4"
 
 For more information, see the [Python tracer installation documentation][2].
 
-## Instrumenting your pytest tests
+## Instrumenting your tests
 
-To enable instrumentation of `pytest` tests, add the `--ddtrace` option when running `pytest`:
+To enable instrumentation of `pytest` tests, add the `--ddtrace` option when running `pytest`, specifiying the name of the service or library under test in the `DD_SERVICE` environment variable, and the environment where tests are being run (i.e. `local` when running tests on a developer workstation, or `ci` when running them on a CI provider) in the `DD_ENV` environment variable:
 
 {{< code-block lang="bash" >}}
-pytest --ddtrace
+DD_SERVICE=my-python-app DD_ENV=ci pytest --ddtrace
 {{< /code-block >}}
 
-You can also add the following configuration to any file used to configure `pytest` (such as `pytest.ini` or `setup.cfg`):
+## Additional configuration settings
 
-{{< code-block lang="ini" >}}
-[pytest]
-ddtrace = 1
-{{< /code-block >}}
+The following is a list of the most important configuration settings that can be used with the tracer, either in code or using environment variables:
+
+`ddtrace.config.service`
+: Name of the service or library under test.<br/>
+**Environment variable**: `DD_SERVICE`<br/>
```

Comment:
It seems like the Python tracer is unique among other tracers to include not only a global configuration setting `DD_SERVICE` but also integration specific configurations including `DD_PYTEST_SERVICE`. 
Our preferred way to set the service name is through the integration specific `DD_PYTEST_SERVICE`, but `DD_SERVICE` should technically be fine. I'll get back to you with more feedback once I discuss with the Python team tomorrow.

Reply:
Ok - I changed it to be consistent with the other tracers, as consistency will mean an easier in-app onboarding flow where we could reuse more between languages. If we need to do something different here is fine, but we should try to have a common way to configure any language.

Reply:
Update from the Python team discussion: We agree it's more important to be consistent with the other tracers, and `DD_SERVICE` should be fine for CIApp customers.

Reply:
Great, thanks for the update @Yun-Kim !